### ISSUE TITLE
`choiceRT_ddm` change the `delta` limit

### ISSUE THREAD

**Issue by YuanboBQ** [CCS-Lab/hBayesDM#95]
Thank you for the great package `hBayesDM`. When using the `choiceRT_ddm`
function to model my choice time data, I found the value of parameter `delta (drift rate)` was limited above zero (>0). However, in many research the `delta` could also a value below zero. So, I want to known how to revised the .stan file to make the value of parameter `delta` range from -Inf to Inf (or -10~10). I try to remove the <lower=0> associated with delta in the .stan file, but it did not work, the estimated delta still above zero.

Another problem is when using the `choiceRT_ddm`
function, if there are some participants only made one kind of choice, the function will not run. Is it possible to fix the problem.

Thanks so much.

**Comment by JaeyeongYang** [comment]
Hi @YuanboBQ, sorry for a late reply.

Thanks for letting us know the issues. For the first issue, I updated the boundaries of the drift rate (`delta`) as `[-Inf, Inf]` in PR #107. The update was merged into the `develop` branch, and it will be released with the version v1.0.0 soon.

For the second issue, please let me know what error messages you got when you ran the codes. That would be much helpful.

**Comment by YuanboBQ** [comment]
Hi @JaeyeongYang, thanks for your reply. I will try your method. For the second issue, it seems there was a problem extract RT data when participants only selected one kind of choice. See the error below:
```
Error in RTu[i, 1:Nu[i]] <- subj_data$rt[subj_data$choice == 2] : 
  replacement has length zero
```

**Comment by JaeyeongYang** [comment]
@YuanboBQ,

It seems to be an error, so I fixed it in PR #109. Could you please re-install this fixed version and confirm if it's working fine with your data? You can reinstall the developmental version by running the following R code:

```R
# If devtools is not installed, you should install it.
if (!require('devtools')) install.packages('devtools')

# Install hBayesDM from Github, on `feature/issue-95-fix-choicert-ddm-preprocess-func` branch
devtools::install_github(
  'CCS-Lab/hBayesDM', subdir = "R",
  ref="feature/issue-95-fix-choicert-ddm-preprocess-func")
```

**Comment by JaeyeongYang** [comment]
For now, I merged the patch into the develop branch, and it will be included in the version 1.0.0 (will be released soon!). If there is any problem even with the updated one, please post an issue again.
